source/getting-started/first-app.md

```diff
@@ -7,28 +6,29 @@ ember new my-new-app
 
 This will create a new `my-new-app` directory with your new Ember app inside.
 
-Once the generation process finishes, launch the Ember CLI development server:
+Once the process finishes, launch the Ember development server:
 
 ```shell
 cd my-new-app
 ember server
 ```
 
-Navigate to `http://localhost:4200` to see your new app in action.
+Navigate to [localhost:4200](http://localhost:4200) to see your app in action.
```

Comment:
Small point, but I think some beginners will be confused when they click this link before they've ever even installed Ember and they expect that it some magic place where Ember apps live.